src/python/visclaw/geoplot.py

```diff
@@ -179,12 +188,18 @@ def surface(current_data):
     q = current_data.q
     h = q[0,:,:]
     eta = q[3,:,:]
-    water = ma.masked_where(h<=drytol, eta)
+
```

Comment:
This does not yield the same object that was previously returned and needs to be made consistent again.